editoast/src/views/pagination.rs

```diff
@@ -75,6 +89,11 @@ pub enum PaginationError {
     #[error("Invalid page size ({page_size}), expected an integer strictly greater than 0")]
     #[editoast_error(status = 400)]
     InvalidPageSize { page_size: i64 },
+    #[error(
+        "Invalid page size ({page_size}), expected an integer lower or equal to {max_page_size}"
+    )]
+    #[editoast_error(status = 400)]
+    MaxPageSize { page_size: i64, max_page_size: i64 },
```

Comment:
```suggestion
    #[error("Invalid page size ({provided_page_size}), expected an integer 0 < page_size <= {max_page_size}")]
    #[editoast_error(status = 400)]
    InvalidPageSize { provided_page_size: i64, max_page_size: i64 },
```

You can merge these two variants.